pkgs/tools/graphics/directx-shader-compiler/default.nix

```
@@ -36,14 +36,14 @@ stdenv.mkDerivation rec {
   installPhase = ''
     mkdir -p $out/bin $out/lib $dev/include
     mv bin/dxc* $out/bin/
-    mv lib/libdxcompiler.so* $out/lib/
+    mv lib/libdxcompiler.so* lib/libdxcompiler.*dylib $out/lib/
```

This will fail because it will either not find any *.so file or any *.dylib file

I suggest reworking this into

```suggestion
    mv lib/libdxcompiler.so* $out/lib/ || :
    mv lib/libdxcompiler.*dylib $out/lib/ || :
```

or something similar

---

It seems to build without issue on both linux and osx though, perhaps null_glob is enabled or something.

---

It seems you are right - https://github.com/NixOS/nixpkgs/blob/b6e98f18950c2b5dbdc725c1181df2ad1be19fee/pkgs/stdenv/generic/setup.sh#L234

Thanks, today I learned something new :-)